CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Drop support for [Go 1.20]. (#4967)
 
+### Deprecated
+
+- Deprecate `go.opentelemetry.io/otel/attribute.Sortable` and methods that use it. (#4734)
```

Comment:
Please include all the other deprecations included in this PR in their own entries as well.